webapp/app/forms/steps/steuerlotse_step.py

```diff
@@ -77,10 +78,10 @@ def _post_handle(self):
             return redirection
         return self.render()
 
-    def _override_session_data(self, stored_data, session_data_identifier=None):
-        if session_data_identifier is None:
-            session_data_identifier = self.session_data_identifier
-        override_session_data(stored_data, session_data_identifier)
+    def _override_storage_data(self, stored_data, data_identifier=None):
+        if data_identifier is None:
+            data_identifier = self.data_identifier
+        self.form_storage.override_data(stored_data, data_identifier)
```

Comment:
S. question for eligibility_step

Reply:
done  -> we need it in the steuerlotse_step because EligibilityStartDisplaySteuerlotseStep needs that functionality.